src/core/Hamiltonian.jl

```diff
@@ -78,10 +79,6 @@ function update_energies!(energies, ham::Hamiltonian, Psi, occupation, ρ=nothin
     ρ === nothing && (ρ = compute_density(ham.basis, Psi, occupation))
 
     energies[:Kinetic] = energy_term_operator(ham.kinetic, Psi, occupation)
-    if ham.pot_external !== nothing
-        dVol = model.unit_cell_volume / prod(basis.fft_size)
-        energies[:PotExternal] = real(sum(real(ρ) .* ham.pot_external) * dVol)
-    end
```

Comment:
You removed this, because it was a duplicate, right?

Reply:
Yes, I forgot to remove it from a previous commit